pkg/exporter/sumologicexporter/sender.go

```diff
@@ -596,11 +596,6 @@ func (s *sender) sendOTLPMetrics(ctx context.Context, md pmetric.Metrics) error
 	for i := 0; i < rms.Len(); i++ {
 		rm := rms.At(i)
 
-		if s.config.TranslateAttributes {
-			translateAttributes(rm.Resource().Attributes()).
-				CopyTo(rm.Resource().Attributes())
-		}
-
 		s.addSourceResourceAttributes(rm.Resource().Attributes())
```

Comment:
Can we move it as well. I don't see a point to make multiple iterations with this change

Reply:
How is the performance change after this pr?